var/spack/repos/builtin/packages/papi/package.py

```diff
@@ -44,6 +93,10 @@ def install(self, spec, prefix):
             # fail, so that PAPI does not get confused
             configure_args.append('MPICC=:')
 
+            configure_args.append(
+                '--with-components={0}'.format(' '.join(
```

Comment:
Could you be explicit about the variants which directly translate to acceptable values for `--with-components`? I think if another variant is added later and it does not relate to this option it could cause a failure here.

Reply:
Currently there are no plans for variants that are not ok values for "--with-components".  If later the is such a variant needed, it will be easy to change this portion of the code to be explicit, but for now I would advocate for keeping it as simple as possible.